docs/core/additional-tools/dotnet-core-uninstall.md

```diff
@@ -0,0 +1,332 @@
+
+### Displaying installed .NET Core SDKs and Runtimes
+
+`dotnet-core-uninstall list` lists installed .NET Core SDKs and Runtimes that can be removed with this tool. Any SDKs or Runtimes that may be required by Visual Studio or SDKs will be displayed with a note of why it is not recommended to uninstall them.
+
+On Windows, the .NET Core Uninstall Tool can only uninstall things that have been installed via the .NET Core SDK or Runtime installer, or the Visual Studio installer prior to Visual Studio 2019 16.3. On macOS, the tool can only uninstall things in the `/usr/local/share/dotnet` folder. This tool does not work on Linux. Because of these limitations, the tool may not be able to uninstall all of the .NET Core SDKs and Runtimes on your machine. You can use `dotnet --info` to find all of the .NET Core SDK's and Runtimes available, including those this tool cannot remove.
+
+#### Synopsis
+
+``` console
+dotnet-core-uninstall list [options]
+```
+
+#### Options
+
+### [Windows](#tab/windows)
+
+* **`--aspnet-runtime`**
+
+  List ASP.NET Core Runtimes.
+
+* **`--hosting-bundle`**
+
+  List .NET Core Runtime & Hosting Bundles.
+
+* **`--runtime`**
+
+  List .NET Core Runtimes.
+
+* **`--sdk`**
+
+  List .NET Core SDKs.
+
+* **`-v, --verbosity <LEVEL>`**
+
+  Set the verbosity level. Allowed values are `q[uiet]`, `m[inimal]`, `n[ormal]`, `d[etailed]`, and `diag[nostic]`.
+
+* **`--x64`**
+
+  List x64 .NET Core SDKs or Runtimes.
+
+* **`--x86`**
+
+  List x86 .NET Core SDKs or Runtimes.
+
+### [macOS](#tab/macos)
+
+* **`--runtime`**
+
+  List .NET Core Runtimes.
+
+* **`--sdk`**
+
+  List .NET Core SDKs.
+
+* **`-v, --verbosity <LEVEL>`**
+
+  Set the verbosity level. Allowed values are `q[uiet]`, `m[inimal]`, `n[ormal]`, `d[etailed]`, and `diag[nostic]`.
+  
+---
+
+#### Examples
+
+* List all .NET Core SDKs and Runtimes that can be removed with this tool:
+
+  ``` console
+  dotnet-core-uninstall list
+  ```
+
+* List all x64 .NET Core SDKs and Runtimes:
+
+  ``` console
+  dotnet-core-uninstall list --x64
+  ```
+
+* List all x86 .NET Core SDKs:
+
+  ``` console
+  dotnet-core-uninstall list --sdk --x86
+  ```
+
+### Uninstalling .NET Core SDKs and Runtimes
+
+`dotnet-core-uninstall dry-run` and `dotnet-core-uninstall whatif` display the .NET Core SDKs and Runtimes that will be removed based on the options provided without performing the uninstall. These commands are equivalent.
+
+Since this is a destructive tool, it is **highly** recommended that you run one of these commands before the remove command to determine the .NET Core SDKs and Runtimes that will be removed. [This article explains which SDKs and Runtimes are safe to remove.](/versions/remove-runtime-sdk-versions#should-i-remove-a-version)
```

Comment:
```suggestion
Since this is a destructive tool, it is **highly** recommended that you do a dry run before the running the remove command. This will tell you what .NET Core SDKs and Runtimes that will be removed. [This article explains which SDKs and Runtimes are safe to remove.](/versions/remove-runtime-sdk-versions#should-i-remove-a-version)
```